### PR TITLE
chore: update googleapis SHA circa 2024-11-28

### PR DESCRIPTION
**Pull request by cuiy0006**
1. Update googleapis SHA based on COMMIT=d675ec222c431e3346ba8aaf0027392fe8b3d90c and COMMIT_DATE=2024-11-28

2. Manually add external dependency `google/iam/v1/resource_policy_member.proto`.

3. Remove some substitutions in `generator/internal/descriptor_utils.cc` to make library generation work.

- - -
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/14871)

### CODE REVIEW

File: generator/internal/descriptor_utils.cc [hunk 1]
```diff
@@ -236,30 +236,32 @@ ParameterCommentSubstitution substitutions[] = {
     //
     // Doxygen thinks these are unrecognized HTML tags. To work around this, we
     // remove the line breaks, so the code quotes occupy a single line.
-    {"agents/\n <Agent ID>", "agents/<Agent ID>"},
-    {"<Agent\n ID>", "<Agent ID>"},
-    {"<Agent ID>/\n testCases/", "<Agent ID>/testCases/"},
-    {"<Agent ID>/\n environments/", "<Agent ID>/environments/"},
+    // {"agents/\n <Agent ID>", "agents/<Agent ID>"},
```
Comment (dbolduc) [inline generator/internal/descriptor_utils.cc:239]: I led you astray. We should delete these lines, not comment them out.

**Review by dbolduc — APPROVED** [review]
Thanks!

File: generator/internal/descriptor_utils.cc (hunk 1 above)
Comment (cuiy0006) [reply generator/internal/descriptor_utils.cc]: Oh, I see. How did you find out which substitution to remove to make the library generation work so quickly?

Comment (dbolduc) [reply generator/internal/descriptor_utils.cc]: It looks like `publish-docs` is still failing.

They changed the comments upstream, but I bet they missed a spot somewhere.

https://github.com/googleapis/googleapis/commit/dc219758b92d84a58e46228cc32654027e31d8b3

There could be more, but it is probably forgetting to escape these format strings. Then `doxygen` sees what it thinks is an HTML tag and freaks out.

https://github.com/googleapis/googleapis/blob/dc219758b92d84a58e46228cc32654027e31d8b3/google/cloud/dialogflow/cx/v3/experiment.proto#L264

We are going to need to add new substitutions to appease `doxygen`. It also good practice to find the offending comments in google3 and send a CL to update them.

Comment (dbolduc) [reply generator/internal/descriptor_utils.cc]: > These iterations are not fun.

Tip: Generating the documentation for all the libraries is sloooooooooow. I would locally set `GOOGLE_CLOUD_CPP_ENABLE=dialogflow_cx` for faster iterations.

https://github.com/googleapis/google-cloud-cpp/blob/5c3ffb5b310cda9cd40107d9c366fe914d925c63/ci/cloudbuild/builds/publish-docs.sh#L45

Comment (cuiy0006) [reply generator/internal/descriptor_utils.cc]: Thank you! That is very clear.

**Review by dbolduc — APPROVED** [review]

**Pull request merged**